src/librustdoc/clean/mod.rs

```diff
@@ -53,51 +55,71 @@ use html::item_type::ItemType;
 pub mod inline;
 mod simplify;
 
+pub struct CleanContext<'a, 'tcx: 'a>  {
+    doc_cx: &'a DocContext<'a, 'tcx>,
+    item_cx: ItemCtxt<'a, 'tcx>,
```

Comment:
This is... unnecessary? Same thing as before, we *really* don't need to inflate the diff like this.